phoenix-core/src/main/java/org/apache/phoenix/query/AdminUtilWithFallback.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.phoenix.query;
```

Comment:
@virajjasani - this needs an Apache license

Reply:
Oh what a miss :(
Thanks @gjacoby126